src/components/DisplayArticle.jsx

```diff
@@ -1,33 +1,31 @@
-import React, { useEffect, useState } from "react";
+import React, { useEffect } from "react";
 import { useParams } from "react-router-dom";
 import { getArticles } from "../modules/getArticles";
 import { Container } from "semantic-ui-react";
+import { useDispatch, useSelector } from "react-redux";
 
 const DisplayArticle = () => {
-  const [article, setArticle] = useState();
-  const [message, setMessage] = useState();
+  const dispatch = useDispatch();
+  const viewArticle = useSelector((state) => state.specificArticle);
+  const errorMessage = useSelector((state) => state.errorMessage);
```

Comment:
You can keep it as it is, or shorten the two lines above to one using object destructuring to access the properties. Like this:

```js
const { specificArticle, errorMessage } = useSelector(state => state);
```
short and neat...

Reply:
Fixed